### PR TITLE
package will now be listed, sorted by lastupdatedtime

### PR DESCRIPTION
**Pull request by viveksinghggits**
This change fixes https://github.com/fission/fission/issues/1093

---
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1334)

### CODE REVIEW

File: pkg/fission-cli/package.go [hunk 1]
```diff
@@ -614,19 +622,21 @@ func createPackage(c *cli.Context, client *client.Client, pkgNamespace string, e
 	if len(pkgName) == 0 {
 		pkgName = strings.ToLower(uuid.NewV4().String())
 	}
+	fmt.Println(time.Now().UTC())
```
Comment (life1347) [inline pkg/fission-cli/package.go]: Is it necessary? If not, we can remove it.

File: pkg/fission-cli/package.go [hunk 2]
```diff
@@ -209,7 +211,8 @@ func updatePackage(client *client.Client, pkg *fv1.Package, envName, envNamespac
 	if needToBuild || forceRebuild {
 		// change into pending state to trigger package build
 		pkg.Status = fv1.PackageStatus{
-			BuildStatus: fv1.BuildStatusPending,
+			BuildStatus:         fv1.BuildStatusPending,
+			LastUpdateTimestamp: time.Now().UTC(),
```
Comment (life1347) [inline pkg/fission-cli/package.go]: You also need to update this function https://github.com/fission/fission/blob/b47b3ecb702a1eb7639cca2dc447fec2edc072ca/pkg/buildermgr/common.go#L126-L129

File: pkg/apis/fission.io/v1/types.go [hunk 3]
```diff
@@ -280,6 +282,9 @@ type (
 
 		// BuildLog stores build log during the compilation.
 		BuildLog string `json:"buildlog,omitempty"` // output of the build (errors etc)
+
+		// will store the timestamp the package was last updated
+		LastUpdateTimestamp time.Time `json:"lastUpdateTimestamp,omitempty"`
```
[the remark this answers is not in the document]
Comment (viveksinghggits) [reply pkg/apis/fission.io/v1/types.go]: corrected the comment.

File: pkg/fission-cli/package.go (hunk 2 above)
Comment (viveksinghggits) [reply pkg/fission-cli/package.go]: added the `LastUpdateTimestamp`

**Comment by viveksinghggits** [comment]
Hi @life1347 
I have commited the new changes.

**Comment by life1347** [comment]
LGTM

**Comment by life1347** [comment]
@viveksinghggits I just opened a PR (https://github.com/fission/fission/pull/1364) to resolve the problem. Thanks for your feedback and I will be more careful next time when reviewing the PR.

**Pull request merged**